apps/ledger-live-mobile/src/screens/SyncOnboarding/index.tsx

```diff
@@ -185,14 +249,20 @@ export const SyncOnboarding = ({ navigation, route }: SyncOnboardingScreenProps)
       drawer: "Could not connect to Stax",
     });
     // handleDesyncClose is then called once the drawer is fully closed
-    setDesyncDrawerOpen(false);
+    setIsDesyncDrawerOpen(false);
   }, []);
 
   const handleDesyncClose = useCallback(() => {
-    setDesyncDrawerOpen(false);
+    setIsDesyncDrawerOpen(false);
     navigation.goBack();
   }, [navigation]);
 
+  const handleAutoRepairClose = useCallback(() => {
+    setIsAutoRepairOpen(false);
+    setIsPollingOn(true);
+    // TODO: reset polling ?
```

Comment:
leftover TODO ?

Reply:
Only forgot to remove the comment on this one